cmd/connector/validate/account_read.go

```diff
@@ -18,20 +20,31 @@ var accountReadChecks = []Check{
 			"std:account:read",
 			"std:account:list",
 		},
-		Run: func(ctx context.Context, spec *connclient.ConnSpec, cc *connclient.ConnClient, res *CheckResult) {
+		Run: func(ctx context.Context, spec *connclient.ConnSpec, cc *connclient.ConnClient, res *CheckResult, readLimit int64) {
 			accounts, _, _, err := cc.AccountList(ctx, nil, nil, nil)
 			if err != nil {
 				res.err(err)
 				return
 			}
+			if len(accounts) == 0 {
+				res.warnf("no accounts")
+				return
+			}
 
+			rand.Shuffle(len(accounts), func(i, j int) {
+				accounts[i], accounts[j] = accounts[j], accounts[i]
+			})
+			count := int64(0)
 			for _, account := range accounts {
+				if count == readLimit {
```

Comment:
Break when equal since we will increment after read the account

Reply:
Instead of manually tracking the count here you can use the first value provided by range:

```
for index, account := range accounts {
  if index+1 == readLimit {
    break
  }
}
```

Reply:
Done, but index for array in Go starting from 0 instead of 1 so I set it to be if `index == readLimit`